css/style.css

```diff
@@ -4,16 +4,118 @@
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    background-color: white;
+    
+}
+.top-navbar {
+    position: fixed;
+    width: 100%;
+    background-color: #fff;
+    z-index: 9999;
+    border-bottom: 2px solid #838994;
+    height: 70px;
+    color: #838994;
+}
+
+.top-navbar .logo {
+    flex-direction: row;
+    margin-right:180px; 
+}
+
+.top-navbar .links {
+    display: flex;
+    flex-direction: row;
+    justify-content: center; 
+    list-style-type: none;
+    margin-left: 200px;
+    margin-right: 200px;  
+    font-size: 17px;
+}
+
+.top-navbar .links li {
+    display: inline;
+    margin-left: 1.5rem;
+}
+
+.top-navbar .first-link {
+    font-weight: bolder;
+    color: #3d4147
+}
+
+.middle-content {
+    background-image: url('../img/first-background.jpg');
+    background-position: center bottom;
+    background-size: cover;
+    height: 750px;
+    text-align: center;
+    color: white;
+}
+
+.middle-content h1 {
+    margin-top: 280px;
+    font-size: 50px;
+    font-weight: lighter;
+}
+
+.middle-content h3 {
+    margin-top: 20;
+    font-size: 25px;
+    font-weight: lighter;
 }
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+button {
+    background-color: #f15a29;
+    color: white;
+    border-style: none; 
+    border-radius: 4px;
+    padding: 16px 24px;
+    font-size: 18px;
+ }
 
+/* SECTION MIDDLE-PAGE / Icons (Devices, Boost, Refill) */
 
+.middle-page .content {
+    text-align: center;
+    color: rgb(29, 26, 26);
+    margin-top: 100px;
+    padding: none;
+}
+
+.middle-page h2 {
```

Comment:
These can be auto-formatted by doing this:
https://stackoverflow.com/a/29973358